config/config.go

```diff
@@ -111,6 +111,7 @@ type GatewayConfig struct {
 	UnhealthyMemberTTL            *time.Duration    `json:"-"`
 	RemoveMemberTimeout           *time.Duration    `json:"-"`
 	AdditionalDimensions          map[string]string `json:",omitempty"`
+	InternalMetrics               *string           `json:",omitempty"`
```

Comment:
Should this be `InternalMetricsListenAddr`? Current name sounds like it's a boolean config flag.